.github/workflows/locale.yml

```diff
@@ -290,3 +290,24 @@ jobs:
         run: |
           make check
           if grep "DEPRECATED" Testing/Temporary/LastTest.log; then exit 1; fi
+  linux-da-dk:
+    name: "linux, da-dk"
+    runs-on: "ubuntu-20.04"
+    env:
+      LANG: "da_DK.UTF-8"
+    steps:
+      - uses: actions/checkout@v3
+      - name: Configure
+        run: |
+          cmake .
+      - name: Check Language
+        run: |
+          grep -q "STUMPLESS_LANGUAGE \"da_DK\"" include/stumpless/config.h
```

Comment:
Sorry I missed this on the first time through: this grep should look for "da-DK" (note the '-' instead of the '_'), which should resolve the CI failure.